pkg/daemon/metrics.go

```diff
@@ -34,12 +34,12 @@ var (
 			Help: "errors from failed drain",
 		}, []string{"node", "err"})
 
-	// MCDPivotErr shows errors encountered during pivot
+	// MCDPivotErr flags error encountered during pivot
 	MCDPivotErr = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
```

Comment:
Looks like this is never decremented so maybe this can be
```suggestion
	MCDPivotErr = prometheus.NewCounterVec(
		prometheus.CounterOpts{
```